src/components/core/settings-header/TemplatesSettings.tsx

```diff
@@ -1,6 +1,48 @@
 import React, { useState } from "react";
 import { Popover, Button } from "antd";
+import styled from "styled-components";
 import { useTemplates } from "../../stores/settings.store";
+import Professional from "../../assets/professional.jpg";
+import Legacy from "../../assets/legacy.jpg";
+
+const TemplateRadio = styled.input.attrs({ type: "radio" })`
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+`;
+
+const TemplateThumbnailImg = styled.img`
+  max-width: 100%;
+  object-fit: cover;
+  height: auto;
+  border: solid 2px transparent;
+`;
+
+const TemplateThumbnail = styled.label`
+  width: 169px;
+  height: 240px;
+  object-fit: cover;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  align-items: center;
+  cursor: pointer;
+  -webkit-user-select: none;
```

Comment:
auto prefixing happens during the build phase I think using POST CSS. Have a look